src/compiler/transformers/component-native/add-define-custom-element-function.ts

```diff
@@ -55,6 +57,13 @@ export const addDefineCustomElementFunctions = (
 
         setupComponentDependencies(moduleFile, components, newStatements, caseStatements, tagNames);
         addDefineCustomElementFunction(tagNames, newStatements, caseStatements);
+
+        if (config.extras?.autoDefineCustomElements) {
```

Comment:
Hey, I wonder if it might make more sense for this to go into the OutputTargetDistCustomElements interface as opposed to config extras?

```suggestion
        import { isOutputTargetDistCustomElements } from 'src/compiler/output-targets/output-utils';
        ...
        if (config.outputTargets.find(isOutputTargetDistCustomElements).autoDefineCustomElements) {
```